lms/djangoapps/courseware/views/views.py

```diff
@@ -994,6 +1008,10 @@ def course_about(request, course_id):
             # context. This value is therefor explicitly set to render the appropriate header.
             'disable_courseware_header': True,
             'pre_requisite_courses': pre_requisite_courses,
+            'course_requirements': course_requirements if course_requirements else None,
+            'student_not_enrollment_in_requirement': \
+                skus_not_enrollment_in_requirements.get(course.id) if skus_not_enrollment_in_requirements else None,
+            "ecommerce_payment_page": EcommerceService().payment_page_url(),
```

Comment:
Why do you need this? @anfbermudezme

Reply:
To pass the context variable to the theme.

![image](https://user-images.githubusercontent.com/30726391/153625994-877c9365-7a66-4e77-9408-9b56f02e4549.png)